app/main/views/services.py

```diff
@@ -95,18 +109,22 @@ def remove_service(service_id):
             'updated_service_name': updated_service.get('serviceName')
         }, 'remove_service')
 
-        return redirect(url_for(".list_services"))
+        return redirect(url_for(".list_services", framework_slug=service["frameworkSlug"]))
 
     return redirect(url_for(
         ".edit_service",
         service_id=service_id,
+        framework_slug=service["frameworkSlug"],
```

Comment:
Why not just use `framework_slug` instead of pulling it out of the service? Same with like 112.